core/src/main/java/de/bwaldvogel/mongo/backend/Utils.java

```diff
@@ -452,4 +465,8 @@ static Document cursorResponse(String ns, List<Document> firstBatch) {
         markOkay(response);
         return response;
     }
+
+    static boolean isNullOrEmpty(String str) {
```

Comment:
i couldn't find an existing version of this anywhere. let me know if i missed one

Reply:
Right, it doesn’t exist yet. But we probably don’t need it (see comment above)